### DISCUSSION TITLE
Please Help Me Understand This Place

### DISCUSSION THREAD

**Discussion by NeilS007** [noDRM/DeDRM_tools#276] [General]
Well, I am not a programmer at all. I went to Apprentice Harper's old site as I realized I had not downloaded new tools in a long time. But I saw that he was no longer supporting the tools and was directed here. Wow. This is really beyond my level of expertise. Used to be downloading the tools was very simple. I mean, the top link gave you the compressed file of tools. Stumbling around reading here I think says that the new tools start with 10 instead of 7? But I cannot find out how to get them. I am really, truly lost. I have Calibre on my Mac and would like the latest tools. How, plesase, can I get them?????

**Comment by MegMM** [comment]
Instead of replying with just the link, here's how to find it from https://github.com/noDRM/DeDRM_tools. On the right-hand side is a section called Releases. Click any link in that section. Scroll down to the bottom for the `.zip`.

**Comment by romad275** [comment]
I did not see a minimum version of Calibre in the version 10 change logs, just that it was fixed to work with Calibre 6. Does the version 10 work with ANY version of Calibre?

Sincerely,
Dennis B. Swaney

Cogito Ergo Mac

**Reply by ElleKayEm** [reply]
The main page says: 

> The v10.0.0 versions of this plugin should both work with Calibre 5.x (Python 3) as well as Calibre 4.x and lower (Python 2). If you encounter issues with this plugin in Calibre 4.x or lower, please open a bug report.

**Comment by romad275** [comment]
Thanks. My current iMac can’t go past Calibre 3.48 so I’m looking at various options. Eventually I’ll have to get a newer iMac but wanted to see what might still work with my current Calibre.

Sincerely,
Dennis B. Swaney

Cogito Ergo Mac

**Reply by ElleKayEm** [reply]
If 10.0.3 doesn't work, then try 6.8.1: https://github.com/apprenticeharper/DeDRM_tools/releases/tag/v6.8.1
